### DISCUSSION TITLE
Overlay settings

### DISCUSSION THREAD

**Discussion by iKingCold** [zaafar/ClickableTransparentOverlay#58] [Q&A] [answered]
Hey, 

Currently trying out your Nuget and it's been working great.
Rendering ImGui on top works really good but I have a couple of questions. 

How can I change the overlay color? If I want it to be black instead of transparent? 
How can I change the windowTitle of the overlay?

**Comment by zaafar** [comment, marked as answer]
Window title is passed in construction of the overlay class.

mad the name suggests, it’s transparent overlay, so anything other than that isn’t supported.
